src/lib/form/schemas/bridge.ts

```diff
@@ -80,7 +80,7 @@ const bridgeRedeemSchema = (params: BridgeRedeemValidationParams): yup.ObjectSch
       is: (isManualVault: string) => isManualVault,
       then: (schema) => schema.required(i18n.t('forms.please_select_your_field', { field: 'redeem vault' }))
     }),
-    [BRIDGE_REDEEM_ADDRESS]: yup.string().required(i18n.t('forms.please_enter_your_field', { field: 'BTC address' })),
+    [BRIDGE_REDEEM_ADDRESS]: yup.string().required(i18n.t('forms.please_enter_your_field')).address(AddressType.BTC),
```

Comment:
This should still contain  ` { field: 'BTC address' }`, otherwise the message is incomplete:  { field: 'BTC address' }))

Reply:
![image](https://github.com/interlay/interbtc-ui/assets/47864599/57f9443b-63e9-469f-9f9b-a0ca96deebea)